im3d.cpp

```diff
@@ -619,6 +619,86 @@ void Im3d::DrawArrow(const Vec3& _start, const Vec3& _end, float _headLength, fl
 	ctx.end();
 }
 
+void Im3d::DrawCone(const Vec3& _origin, const Vec3& _normal,float height, float _radius, int _detail){
+
+    Context& ctx = GetContext();
+    #if IM3D_CULL_PRIMITIVES
+        if (!ctx.isVisible(_origin, _radius))
```

Comment:
I think this should be the bounding sphere of the cone: `_origin + _normal * height / 2`, `height / 2`?